api/localdev.ps1

```diff
@@ -0,0 +1,6 @@
+# Environment variable settings needed to run API locally via npm run start:api
```

Comment:
Please use env files, not .ps1.  Our only supported platform is Linux.

Reply:
Powershell script is removed.